modular_skyrat/modules/basic_mobs/code/chinchilla.dm

```diff
@@ -0,0 +1,100 @@
+/mob/living/basic/pet/chinchilla
+	name = "chinchilla"
+	desc = "They're a chinchilla."
```

Comment:
I feel like we can get something a little fancier/funnier than that for a desc, nah?